conda_forge_tick/migrate_update_version.py

```diff
@@ -0,0 +1,56 @@
+# type this code in at the interpreter
+from conda_forge_tick.utils import load_graph
+from conda_forge_tick.update_upstream_versions import (
+    get_latest_version,
+    PyPI,
+    CRAN,
+    NPM,
+    ROSDistro,
+    RawURL,
+    Github,
+)
+import random
+import json
+
+sources = (PyPI(), CRAN(), NPM(), ROSDistro(), RawURL(), Github())
+
+print("Reading graph")
+gx = load_graph()
+_all_nodes = [t for t in gx.nodes.items()]
+random.shuffle(_all_nodes)
+
+# from here you can inspect the graph object, and what node will be updated:
+print(f"Number of nodes: {len(gx.nodes)}")
+Node_count = 0
+
+to_update = {}
+to_update["nodes"] = []
+for node, node_attrs in _all_nodes:
```

Comment:
Now i am using the `_all_nodes`

Reply:
I still need to specify an dir to write the .json file. (But i don't know where)